src/pages/basic-management/employee/add/Index.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent, Fragment } from 'react';
+import { connect } from 'dva';
+import router from 'umi/router';
+import {
+    List,
+    Card,
+    Row,
+    Col,
+    Radio,
+    Input,
+    InputNumber,
+    Progress,
+    Button,
+    Icon,
+    Dropdown,
+    Menu,
+    Avatar,
+    Modal,
+    Form,
+    DatePicker,
+    Select,
+    Cascader,
+    message,
+    Divider,
+} from 'antd';
+import PageHeaderWrapper from '@/components/PageHeaderWrapper';
+import styles from '../../../List/TableList.less';
+
+const FormItem = Form.Item;
+const { Option } = Select;
+
+@connect(({ employee, depart, loading }) => ({
+    employee,
+    depart,
+    loading: loading.models.employee,
+}))
+@Form.create()
+class employeeAdd extends PureComponent {
+    state = {};
+
+    componentDidMount = () => {
+        const { dispatch } = this.props;
+        dispatch({
+            type: 'depart/fetch',
+            payload: {}
+        })
+    };
+
+    handleSubmit = e => {
+        e.preventDefault();
+        this.props.form.validateFieldsAndScroll((err, values) => {
+            if (!err) {
+                const { dispatch } = this.props;
+                if (!values.departId) {
+                } else {
+                    values.departId = values.departId[values.departId.length - 1];
+                }
+                dispatch({
+                    type: 'employee/add',
+                    payload: values,
+                });
+            }
+        });
+    };
+
+    handleCancle = e => {
+        router.push('/basic-management/employee/list');
+    };
+
+    render() {
+        const {
+            form: { getFieldDecorator },
+            depart: { data }
+        } = this.props;
+        const formItemLayout = {
+            labelCol: {
+                xs: { span: 24 },
+                sm: { span: 4 },
+            },
+            wrapperCol: {
+                xs: { span: 24 },
+                sm: { span: 8 },
+            },
+        };
+        const tailFormItemLayout = {
+            wrapperCol: {
+                xs: {
+                    span: 24,
+                    offset: 0,
+                },
+                sm: {
+                    span: 16,
+                    offset: 4,
+                },
+            },
+        };
+        return (
+            <PageHeaderWrapper>
+                <Card bordered={false}>
+                    <div className={styles.tableList}>
+                        <Form {...formItemLayout} onSubmit={this.handleSubmit} labelAlign="right">
```

Comment:
Codacy found an issue: [Prop spreading is forbidden](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)